### DISCUSSION TITLE
How can I help in Arabic Translation?

### DISCUSSION THREAD

**Discussion by izada** [aonez/Keka#1514] [Ideas]
I didn't find Arabic in the app. How can I translate the app? Any XML or other file for translation?

**Comment by aonez** [comment]
Hi @izada! The [iOS version is already translated](https://github.com/aonez/Keka/tree/master/Translations-iOS/ar.xcloc), you can check out the files and the [README](https://github.com/aonez/Keka/blob/master/Translations-macOS/README.md) for the macOS version here: https://github.com/aonez/Keka/tree/master/Translations-macOS

Let me know if you need anything else.

**Reply by izada** [reply]
I didn't find Arabic in macOS, can I download files and create ar.lproj and send them to you by email or so to commit?

**Reply by aonez** [reply]
Sure @izada, just download the files here: https://github.com/aonez/Keka/archive/refs/tags/v1.4.4.tar.gz

Use the `Translations-macOS)/en.lproj/` as your source and upload it here or send it via mail to info@keka.io.

Many thanks in advance 👍

**Reply by izada** [reply]
Thanks, the files are done, and details shared by email ✅

**Reply by aonez** [reply]
Thanks a lot @izada!! The translation is already in the repo:
https://github.com/aonez/Keka/commit/d38d130076ad9d5afe0968ffd85530d7f56031fa

Just had to revert the order of this string that was ar-en instead of en-ar:
https://github.com/aonez/Keka/commit/d38d130076ad9d5afe0968ffd85530d7f56031fa#diff-0d4135ce32d88fc6d5adcf6036e00b4ac91aa7dad24891cc5dfbf72491f4caebR133

**Reply by aonez** [reply]
I'll create a build tomorrow so you can try it.

**Reply by izada** [reply]
I am ready to test it tomorrow. Thanks

**Comment by aonez** [comment]
Here the build containing this translation:
https://github.com/aonez/Keka/releases/download/dev-test-builds/Keka-v1.4.5.r5489.7z

Thanks again @izada

**Reply by izada** [reply]
Looks working 👍 Thanks
